.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+
+Please run down the following list and make sure you've tried the usual "quick fixes":
+
+  - Search the issues already opened: https://github.com/googleapis/gapic-generator-java/issues
+  - Check for answers on StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform
+
+If you are still having issues, please include as much information as possible:
+
+#### Environment details
+
+1. Specify the API at the beginning of the title. For example, "BigQuery: ...").
```

Comment:
Do we still need to specify API for issues in `gapic-generator-java`? It makes sense for the SDK monorepo, but maybe something like `Specify the module` makes more sense for this repo? Or maybe we don't need this step? Since technically all submodules are core components of the generator and we don't have to tell the differences?